### ISSUE TITLE
Youtube Link

### ISSUE THREAD

**Issue by garrettgregor** [rishavanand/github-profilinator#145]
**Describe the bug**
Youtube link broken

**To Reproduce**

1. Go to add youtube link in connect with me in template 1
2. Type in username
3. Click on youtube link in preview
4. See page not found

**Expected behavior**
Expect page to go to youtube.com/username, instead goes to youtube.com/user/username

**Screenshots**

**Desktop (please complete the following information):**

-   OS: MacOS
-   Browser Chrome
-   Version 115

**Comment by RafaGomezGuillen** [comment]
Fixed in #153

**Comment by garrettgregor** [comment]
Nice one! I forgot I had even submitted this 😆

**Comment by RafaGomezGuillen** [comment]
Yeah!! So much time has passed since Aug 21, 2023. Lets see if PR is merged!
